gridcapa-core-valid-app/src/main/java/com/farao_community/farao/gridcapa_core_valid/app/study_point/StudyPointService.java

```diff
@@ -58,9 +64,11 @@ public StudyPointResult computeStudyPoint(StudyPoint studyPoint, Network network
             Map<String, InitGenerator> initGenerators = setPminPmaxToDefaultValue(network, scalableZonalData);
             NetPositionsHandler.shiftNetPositionToStudyPoint(network, studyPoint, scalableZonalData, coreNetPositions);
             resetInitialPminPmax(network, scalableZonalData, initGenerators);
-            String url = saveShiftedCgm(network, studyPoint);
+            String cgmUrl = saveShiftedCgm(network, studyPoint);
+            RaoRequest raoRequest = buildRaoRequest(coreValidRequest.getId(), coreValidRequest.getCbcora().getUrl(), cgmUrl, coreValidRequest.getGlsk().getUrl());
+            RaoResponse raoResponse = raoRunner.runRao(raoRequest);
```

Comment:
It's better to add a try catch to catch any RuntimeException could be occured during Rao running. It could be interesting to add logs when the Rao is sent or received